financial-connections/src/main/java/com/stripe/android/financialconnections/features/manualentrysuccess/ManualEntrySuccessViewModel.kt

```diff
@@ -57,5 +76,15 @@ internal class ManualEntrySuccessViewModel @Inject constructor(
 }
 
 internal data class ManualEntrySuccessState(
-    val completeSession: Async<FinancialConnectionsSession> = Uninitialized
-) : MavericksState
+    val args: Bundle?,
```

Comment:
Why exactly is this needed here? If it’s just used in `ManualEntrySuccessViewModel`, can we store the last-4 here directly?

Reply:
https://github.com/stripe/stripe-android/pull/7761/commits/073cbe74fcc68cfb1b29023c6f359ae90670eb8f